test/macros.jl

```diff
@@ -106,6 +134,10 @@ end
     )
 end
 
+@testset "testitem macro runs immediately outside `runtests`" begin
+    @test nothing == @testitem "run" begin; @test true; end
+end
```

Comment:
wasn't sure how best to test the new behaviour... suggestions for improvements welcome!

Reply:
I guess you could use Mocking to ensure that the expected testing functions inside ReTestItems runtestitem were called?

Or you could capture the output and test that it printed a test summary at the end?